src/prog_models/models/battery_circuit.py

```diff
@@ -3,6 +3,7 @@
 
 from .. import prognostics_model
 
+import math
```

Comment:
to avoid double importing parts of math, let's remove this and change line 7 to from math import exp, inf. and then change math.inf -> inf. 

Same thing in the other files where parts of math are imported

Reply:
oh okay. I tried this but i left it as math.inf and it wasn't working